addons/hr_recruitment_survey/tests/test_recruitment_survey.py

```diff
@@ -36,13 +56,16 @@ def setUpClass(cls):
         })
 
     def test_send_survey(self):
-        # We ensure that response is False because we don't know test order
-        self.job_sysadmin.response_ids = False
+        # We ensure that response and interviewers are False because we don't know test order
+        self.job_sysadmin.write({'response_ids': False, 'interviewer_ids': False})
         Answer = self.env['survey.user_input']
         answers = Answer.search([('survey_id', '=', self.survey_sysadmin.id)])
         answers.unlink()
 
-        self.survey_sysadmin.write({'access_mode': 'public', 'users_login_required': False})
+        self.survey_sysadmin.write({
+            'access_mode': 'public',
+            'users_login_required': False,
+            'survey_type': 'custom'})
         action = self.job_sysadmin.action_send_survey()
 
         invite_form = Form(self.env[action['res_model']].with_context({
```

Comment:
```suggestion
        invite_form_custom = Form(self.env[action['res_model']].with_context({
```
And keeping the `_custom` suffix on `invite` below (or `custom_type`), it will ease reading.